extensions/adobe/experience/adcloud/advertisementdetails.example.1.json

```diff
@@ -1,9 +1,9 @@
 {
-  "https://ns.adobe.com/experience/adcloud/advertisementdetails/adId": 4116904851,
-  "https://ns.adobe.com/experience/adcloud/advertisementdetails/runtime": "HTML5",
-  "https://ns.adobe.com/experience/adcloud/advertisementdetails/class": "video",
-  "https://ns.adobe.com/experience/adcloud/advertisementdetails/adUnitType": "HTML5",
-  "https://ns.adobe.com/experience/adcloud/advertisementdetails/asset": {
+  "https://ns.adobe.com/experience/adcloud/adId": 4116904851,
```

Comment:
this needs to be a string with the new schema

Reply:
Fixed